MAUI/DataGrid/export-to-pdf.md

```diff
@@ -0,0 +1,665 @@
+
+* `PdfGrid`: Customizes the pdfGrid properties such as `Background`, `CellPadding`, `CellSpacing`, and so on.
+* `PdfRow`: Specifies the `PDFGridRow` to be exported. Customizes the properties of a particular row. 
+* `Record`: Gets the collection of exported underlying data objects.
+* `RowType`: Specifies the row type using `ExportRowType` enum. Checks the row type and applies different styles based on the row type.
+
+You can use this event to customize the properties of the grid rows exported to PDF. The following code example illustrates how to change the background color of the record rows and caption summary rows when exporting.
+
+{% tabs %}
+{% highlight c# %}
+pdfExport.RowExporting += pdfExport_RowExporting; 
+void pdfExport_RowExporting (object sender, DataGridRowPdfExportingEventArgs e)
+{
+    if (e.RowType == ExportRowType.Record) {
+        if ((e.Record.Data as OrderInfo).IsClosed)
+            e.PdfRow.Style.BackgroundBrush = PdfBrushes.Yellow;
+        else
+        e.PdfRow.Style.BackgroundBrush = PdfBrushes.LightGreen;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Customize the grid rows while exporting using the RowExporting event" src="Images\export-to-pdf\maui-datagrid-row-exporting.png" width="689"/>
+
+### CellExporting
+
+The `DataGridCellPdfExportingEventHandler` delegate allows customizing the styles for the header cells, record cells, group caption cells, and group summary cells. The `CellExporting` event is triggered with `DataGridCellPdfExportingEventArgs` that contains the following properties:
+
+The `DataGridCellPdfExportingEventHandler` delegate allows customizing the styles for header cells, record cells, and group caption cells. The `CellExporting` event is triggered with `DataGridCellPdfExportingEventArgs` that contains the following properties:
+
+* `CellType`: Specifies the cell type using `ExportCellType` enum. Checks the cell type and applies different cell styles based on the cell type.
+
+* `CellValue`: Contains the exported actual value to format the PDF using the `Range` property.
+
+* `ColumnName`: Specifies the column name (MappingName) of the exporting cell. Applies formatting for a particular column by checking the `ColumnName`.
+
+* `Handled`: Determines whether the cell is exported to PDF or not.
+
+* `PdfGrid`: Specifies the `PDFGridCell` to be exported. Customizes Background, Foreground, Font, Alignment, etc., properties of a particular cell.
+
+* `Record`: Gets the collection of exported underlying data objects.
+
+You can use this event to customize the properties of the grid cells exported to PDF. The following code example illustrates how to customize the background color, foreground color, and cell value of the header cells, record cells, and caption summary cells when exporting.
```

Comment:
The `DataGridCellPdfExportingEventHandler` delegate allows for customizing the styles of header cells, record cells, and group caption cells. The `CellExporting` event is triggered with `DataGridCellPdfExportingEventArgs`, which contains the following properties:

* `CellType`: Specifies the cell type using the `ExportCellType` enum. It checks the cell type and applies different cell styles based on the type.

* `CellValue`: Contains the actual exported value used to format the PDF using the `Range` property.

* `ColumnName`: Specifies the column name (MappingName) of the exporting cell. It applies formatting for a particular column by checking the `ColumnName`.

* `Handled`: Determines whether the cell is exported to PDF or not.

* `PdfGrid`: Specifies the `PDFGridCell` to be exported. It customizes properties such as Background, Foreground, Font, Alignment, etc., for a particular cell.

* `Record`: Retrieves the collection of underlying data objects being exported.

You can use this event to customize the properties of grid cells that are exported to PDF. The code example below demonstrates how to customize the background color, foreground color, and cell value of header cells, record cells, and caption summary cells during the export process.

Reply:
Updated the content